lib/docs-tools/README.md

```diff
@@ -0,0 +1,6 @@
+# Storybook Docs Utils
+
+Shared utility functions for frameworks to implement docs
+
+- ArgType extraction
+- Dynamic snippet generation
```

Comment:
Perhaps add a comment to this file outlining that it depended on by all the frameworks and should be as minimal as possible.

Reply:
Good point. Added in the other PR